os/mm/kmm_heap/kmm_sem.c

```diff
@@ -93,8 +93,7 @@
  *   OK on success; a negated errno on failure
  *
  ************************************************************************/
-
-int kmm_trysemaphore(void)
+int kmm_trysemaphore(void *dummy_addr)
```

Comment:
Why do you add an argument without any usages?

Reply:
kmm_trysemaphore has multiple definitions based on configs. In some cases, 

`#define kmm_trysemaphore(a)      umm_trysemaphore(a)`

umm_trysemaphore() always requires argument, but kmm_trysemaphore() does not need argument. In order to satisfy above cases, I have added dummy argument to kmm_trysemaphore, so that we can maintain compatibility between kmm_trysemaphore(a) and umm_trysemaphore(a).